Portfolio/index.html

```diff
@@ -1,34 +1,147 @@
+<head>
+    <title>Portfolio</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="index.css" />
+    <link rel="stylesheet" href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css'>     
+</head>
+    
+    <DIV class="topline">
+        <ul>
+            <li><h1>Deo</h1></li>
+        <li><a href="#">Home</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Skills</a></li>
+        <li><a href="#">Reviews</a></li>
+        <li><a href="#">Project</a></li>
+        <li><img class="call"
+            src="https://th.bing.com/th?id=OIP.Wt910bi4UDoTKGbWMw_1RAHaHx&w=244&h=256&c=8&rs=1&qlt=90&o=6&dpr=1.8&pid=3.1&rm=2"
+            height="12px">
+        <input class="button" type="button" onclick="location.href='www.google.com';" value="Contact"></li>
+    </ul>
+    </DIV>
+<div class="john">
+    <ul>
+    <li><h1><bold>John Doe</bold></h1></li>
 
-  <body>
-    <h1>Jenny Doe</h1>
-    <img width = 100 src="https://www.pngjoy.com/pngm/258/4974996_welcome-girl-professional-young-woman-png-hd-png.png" alt="jenny">
-  <h1> Front End <br>Developer</h1>
-  <h4>GET TO KNOW ME</h4>
-  <p>’m a junior front-end developer and like to code in my spare time. <br> Using HTML, CSS, and JavaScript, I have expertise in creating responsive websites. <br> I have also worked with well-known frontend frameworks like React and Vue.js. <br> I prefer to try out new technologies while I’m not working on projects.</p><br>
-<p>NAME: JENNY DOE<br>
-    AGE: 25<br>
-    EXPERIENCE: 2years</p>
-    <h4>SKILLS</h4>
-    <h4>GET IN TOUCH </h4>
-    <h4>CONTACT</h4>
-    <form>
-        <label for="NAME">NAME:</label>
-        <input type="text" id="NAME"  NAME="NAME"><br>
-        <label for="E-MAIL"> E-MAIL:</label>
-        <input type="text" id="E-MAIL"  NAME="E-MAIL:">
-        <form action="/action_page.php">
-            <p><label for="Message">Message:</label></p>
-            <textarea id="Message" name="Message" rows="4" cols="50">ADD CONTENT</textarea>
-            <br>
-            <input type="submit" value="Submit">
-</Form>
+    <li><img class="Doe" src="johnDeo.png"
+        height="150px" width="150px"></li>
+    <li><h1> <bold>Front End <br> Developer</bold></h1></li>
+</ul>
+</div>
+    <h3>GET TO KNOW ME</h3>
+    <p>’m a junior front-end developer and like to code in my spare time. <br> Using HTML, CSS, and JavaScript, I have expertise in creating responsive websites. <br> I have also worked with well-known frontend frameworks like React and Vue.js. <br> I prefer to try out new technologies while I’m not working on projects.
+        ex!<br><br>Name:Jhon Doe<br>Age:25<br>Exeperience:2years</p>
+    <h3>SKILLS</h3>
+    <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcT81jebNZhch3fhN8HiYmNkP76R6B38DzeH4g&usqp=CAU" height="20px"><br><span>html</span><br>
+    <img src="https://creazilla-store.fra1.digitaloceanspaces.com/icons/3247738/css3-alt-icon-md.png" height="20px"><br>
```

Comment:
The <strong> element should be inside a <p> or <span> element for proper semantic markup.